### PR TITLE
Add aditional log line to xem update

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by p0psicles** [comment]
Why do you want to add these logs. What was the issue?

File: medusa/scene_numbering.py [hunk 1]
```diff
@@ -495,17 +493,18 @@ def xem_refresh(indexer_id, indexer, force=False):
             if not indexerApi(indexer).config.get('xem_origin'):
                 return
             # XEM MAP URL
-            url = "http://thexem.de/map/havemap?origin=%s" % indexerApi(indexer).config['xem_origin']
+            url = "http://thexem.de/map/havemap?origin={0}".format(indexerApi(indexer).config['xem_origin'])
             parsedJSON = helpers.getURL(url, session=xem_session, returns='json')
             if not parsedJSON or 'result' not in parsedJSON or 'success' not in parsedJSON['result'] or 'data' not in parsedJSON or str(indexer_id) not in parsedJSON['data']:
+                logger.log(u'No XEM data for show "{0} on {1}"'.format(indexer_id, indexerApi(indexer).name), logger.INFO)
```
Comment (duramato) [inline medusa/scene_numbering.py]: I added this one @p0psicles becouse it was showing when that it was contacting xem but none that there were no results. The other lines I only changed to use .format()

Comment (fernandog) [inline medusa/scene_numbering.py]: it there is not data for the show, why is on XEM?

Comment (duramato) [inline medusa/scene_numbering.py]: Becouse it isn't, it's just querying to see if there is..

Comment (fernandog) [inline medusa/scene_numbering.py]: @duramato logger.log(u'No XEM data for show ID {0} on "{1}"' right?

Comment (duramato) [inline medusa/scene_numbering.py]: I only used the same log that exists below.

**Comment by duramato** [comment]
What this fixes is
Currently it only shows:
```
2016-12-04 18:34:44 DEBUG    Thread-17 :: [88398f1] Looking up XEM scene mapping for show 77262 on TVDBv2
```
With this: 
```
2016-12-04 18:34:44 DEBUG    Thread-17 :: [88398f1] Looking up XEM scene mapping for show 77262 on TVDBv2
```
```

**Comment by medariox** [comment]
Can we avoid the double quotes? We usually use single quotes if at all.

**Comment by duramato** [comment]
Double quotes where used when there where either aposthrophes or a quote within the log

**Pull request merged**